lib/board.rb

```diff
@@ -1,113 +1,101 @@
 class Board
-  attr_accessor :grid 
 
   def initialize(symbols = Array.new(9))
     @grid = symbols
   end
 
   def is_empty
```

Comment:
The ruby convention is to end predicates with a question mark and omit the `is_` prefix. So the preferred name would be `empty?` over `is_empty`